### ISSUE TITLE
javascript 기능 만들기

### ISSUE THREAD

**Issue by hhlvv** [hhlvv/omo#17]
- [ ]  디바이스 확인 후 폰에서만 문자, 전화 가능하기
- [ ]  링크 공유하기
- [ ] 지도 첨부하기

**Comment by jongfeel** [comment]
휴대폰에서 전화, 문자 되는 코드 찾지 않으셨어요?

**Comment by hhlvv** [comment]
네, 근데 지금 화면이 정리가 안되서 기능을 안넣어뒀어요! 레이아웃 잡힌 후에 하려고 합니다.😢 바뀌는게 점점 많아져서요
